src/app/components/Headers/HeaderRecipeDetail.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Card,
+  CardContent,
+  IconButton,
+  ImageListItemBar,
+  Typography,
+} from '@mui/material';
+import { Box } from '@mui/system';
+import StarBorderRoundedIcon from '@mui/icons-material/StarBorderRounded';
+import React from 'react';
+import './index.css';
+
+export default (): JSX.Element => {
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <Box sx={{ width: '100%', height: '200px', overflow: 'hidden' }}>
+        <img
+          className="image-header"
```

Comment:
This is the image of the header, so I'd rather call this e.g. `headerImage`.

Reply:
That's a MUI component, so I can't do anything about the naming 😬